src/test/java/com/gamzabat/algohub/feature/recommendation/domain/GroupActivityWeeklyTest.java

```diff
@@ -0,0 +1,60 @@
+package com.gamzabat.algohub.feature.recommendation.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.gamzabat.algohub.feature.group.studygroup.domain.StudyGroup;
+
+class GroupActivityWeeklyTest {
+
+	@Test
+	@DisplayName("GroupActivityWeekly 엔티티 생성 성공")
+	void createGroupActivityWeekly_Success() {
+		// given
+		LocalDateTime now = LocalDateTime.now();
+		LocalDateTime weekStart = now.minusWeeks(1);
+		LocalDateTime weekEnd = now;
+
+		StudyGroup studyGroup = StudyGroup.builder()
+			.name("테스트 스터디 그룹")
+			.build();
+
+		// when
+		GroupActivityWeekly activity = GroupActivityWeekly.builder()
+			.studyGroup(studyGroup)
+			.weekStart(weekStart)
+			.weekEnd(weekEnd)
+			.submissions(10)
+			.comments(5)
+			.activeScore(17.5)
+			.build();
+
+		// then
+		assertThat(activity.getStudyGroup()).isEqualTo(studyGroup);
+		assertThat(activity.getWeekStart()).isEqualTo(weekStart);
+		assertThat(activity.getWeekEnd()).isEqualTo(weekEnd);
+		assertThat(activity.getSubmissions()).isEqualTo(10);
+		assertThat(activity.getComments()).isEqualTo(5);
+		assertThat(activity.getActiveScore()).isEqualTo(17.5);
```

Comment:
아래처럼 쓰면 실패해도 나머지 assertion은 계속 검증해서 모든 실패를 한 번에 보기 좋습니다!
반영하실 필요는 없고 혹시나 싶어서 참고용으로 코멘트 남깁니다..!

```suggestion
        Assertions.assertAll(
            () -> assertThat(joinInfo.getStudyGroup()).isEqualTo(studyGroup),
            () -> assertThat(joinInfo.getWindowStart()).isEqualTo(windowStart),
            () -> assertThat(joinInfo.getWindowEnd()).isEqualTo(windowEnd),
            () -> assertThat(joinInfo.getNewMembers()).isEqualTo(5),
            () -> assertThat(joinInfo.getMembersBeforeWindow()).isEqualTo(20),
            () -> assertThat(joinInfo.getJoinRate()).isEqualTo(0.25)
        );
```

Reply:
좋은 팁 너무 감사합니다!